src/Physics/physicsHelper.ts

```diff
@@ -174,62 +173,67 @@ export class PhysicsHelper {
             return null;
         }
 
-        var event = new PhysicsVortexEvent(this._scene, origin, radius, strength, height);
+        var event = new PhysicsVortexEvent(this, this._scene, origin, eventOptions);
 
         event.dispose(false);
 
         return event;
     }
 }
 
+export class PhysicsHelperOptions {
+    /**
+     * If set to true, it will use the native raycasting
+     */
+    public useNativePhysicsRaycastingIfAvailable: boolean = true;

```

Comment:
I would prefer to have that at plugin level so users do not have to directly know about it (so each plugin can define if it supports it or not)